### PR TITLE
homepage and setting up dev area for roger

### PR DESCRIPTION
**Pull request by amberRrucker**
Homepage styling and creating footer dev area for Roger.

### CODE REVIEW

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 1]
```diff
@@ -1,41 +1,54 @@
...
+      hr{ border-color:$mediumteal; }
+      h3, a{ color:$mediumteal; }
     }
   }
 }
 
+#solutions{
+  a{ text-decoration:none; }
+  .solution:hover{
+   .border-white{
```
Comment (SyntaxStacks) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: indentation is off a bit

**Comment by SyntaxStacks** [comment]
+1

File: styleguide/derek/_partials/_footer.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<!--[footernav]-->
+<nav class="">
```
Comment (bighappyface) [inline styleguide/derek/_partials/_footer.html]: This should be migrated into `_footer.jade` as 100% Jade or into an EJS file to favor long-form HTML. Either way this kind of split isn't very desirable.

Comment (amberRrucker) [inline styleguide/derek/_partials/_footer.html]: Sure thing. Do you want to do the same for the ceiling and main nav @bighappyface ?

Comment (bighappyface) [inline styleguide/derek/_partials/_footer.html]: May as well

**Pull request merged**